src/services/doppler-legacy-client.ts

```diff
@@ -341,7 +350,6 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
       retryDelay: (retryCount: any, error: any) => {
         addLogEntry({
           origin: window.location.origin,
-          section: 'Login/legacyClient',
```

Comment:
Is it not necessary anymore?

Reply:
We cannot have all logs with the same section, either remove it or replace it. Also this informatiton is a bit redundant, see in the log:

![image](https://user-images.githubusercontent.com/2439363/79222545-1a1e2500-7e2e-11ea-9b38-27cf8a0d1ac5.png)

Here we have section and then in request information we have also the url.